### ISSUE TITLE
[Bug]: Chipped Express doesn't work for blocks from Chisel Chipped Integration 

### ISSUE THREAD

**Issue by Jaymus3** [ThePansmith/Monifactory#59]
### Issue Summary

Items from Chisel Chipped Integration do not appear to be working correctly with Chipped Express, and only the base variants of those blocks are currently craftable as a result.

### Reproduction steps

1.  Try to craft "Large Tiles" laboratory block
2. You can't craft it.

**Comment by ThePansmith** [comment]
![image](https://github.com/ThePansmith/Monifactory/assets/70342772/c529b7f5-3226-4aae-aecc-f0d78a5ff271)
?

**Comment by Jaymus3** [comment]
This issue appears to only be present when playing in multiplayer.  I just tested a single player world and I can see the stonecutter recipes but when connected to the server I don't have them.  I suspected I may have modified the configuration so I verified with a clean copy of the server to be sure.  Screenshot from the server:
![CleanShot 2024-04-20 at 09 57 42](https://github.com/ThePansmith/Monifactory/assets/1351081/a1097a42-5fdc-4e40-be26-461bca2b1c10)


Screenshot from single player:
![CleanShot 2024-04-20 at 09 59 13](https://github.com/ThePansmith/Monifactory/assets/1351081/0ed30bc6-af20-4fd6-975b-4171f722815f)
